roles/edpm_telemetry/templates/ceilometer_agent_compute.json.j2

```diff
@@ -16,10 +16,10 @@
         "/etc/hosts:/etc/hosts:ro",
         "/etc/pki/tls/certs/ca-bundle.trust.crt:/etc/pki/tls/certs/ca-bundle.trust.crt:ro",
         "/etc/localtime:/etc/localtime:ro",
-        "/etc/pki/ca-trust/extracted:/etc/pki/ca-trust/extracted:ro",
         "/etc/pki/ca-trust/source/anchors:/etc/pki/ca-trust/source/anchors:ro",
-        "/dev/log:/dev/log",
-        "/etc/pki/tls/cert.pem:/etc/pki/tls/cert.pem:ro",
-        "/etc/pki/tls/certs/ca-bundle.crt:/etc/pki/tls/certs/ca-bundle.crt:ro"
+{% if edpm_telemetry_tls_certs_enabled %}
```

Comment:
technically should always mount this if it exists. Could have a custom TLS CA bundle without having internal TLS enabled (e.g need to connect to an external service using a custom CA). However most edpm roles are currently like this so can follow-up with a PR to fix it everywhere. Also expect we need to change dataplane operator logic for this.